lib/graphql/queries.js

```diff
@@ -748,3 +748,23 @@ export const hostPendingApplicationsQuery = gql`
 export const addCollectiveCoverData = (component, options) => {
   return graphql(collectiveCoverQuery, options)(component);
 };
+
+export const invoicesQuery = gql`
```

Comment:
Would be great if the query could be API v2.

Generally, we avoid putting queries in this file. It's better to put them closer to where they used. And `export` them if they need to be shared somewhere.